src/parameterized_tendencies/gravity_wave_drag/non_orographic_gravity_wave.jl

```diff
@@ -388,3 +446,7 @@ end
 function calc_intermitency(ρ_source_level, source_ampl, nk, Bsum)
     return (source_ampl / ρ_source_level / nk) / Bsum
 end
+
+@inline reduce_fun1(a, b) = ifelse(a[1] < b[1], a, b)
+@inline reduce_fun2(a, b) = ifelse(b[1] < 0, a, b)
```

Comment:
```suggestion
@inline reduce_fun2(a, b) = ifelse(b[1] <= 0, a, b)
```
(I think this is exactly the same as the previous logic, although it probably doesn't matter much)